dev/bench/bench.sh

```diff
@@ -433,10 +436,18 @@ for coq_opam_package in $sorted_coq_opam_packages; do
     export COQ_OPAM_PACKAGE=$coq_opam_package
     if [ ! -z "$BENCH_DEBUG" ]; then
         opam list
-        opam show $coq_opam_package || continue 2
+        opam show $coq_opam_package || {
+            failed_packages="$failed_packages
+$coq_opam_package (unknown package)"
```

Comment:
Please use `$'\n'` instead of inserting a newline in the script. We're already asking for bash.

Reply:
All right. This is horrible to my eyes but YMMV.

Reply:
Why not
```bash
failed_packages="$(printf "%s\n%s (unknown package)" "$failed_packages" "$coq_opam_package")"
```
?

Reply:
Or stick `newline=$'\n'` at the top of the file and use `$newline` here